manim/mobject/svg/text_mobject.py

```diff
@@ -1066,7 +1065,7 @@ def __init__(
         else:
             self.line_spacing = self._font_size + self._font_size * self.line_spacing
 
-        file_name = self.text2svg()
+        file_name = self.text2svg(Color(color) if color else None)
```

Comment:
What happens if `None` is passed here? I'm not familiar with manimpango so I'm not sure if None could cause problems here

Reply:
Since `None` is not a valid color it would default to black, but I've made it more explicit.